pages/1.11/security/ent/secrets/create-secrets/index.md

```diff
@@ -90,7 +108,7 @@ The contents of the file (referred to below as `my-secret.txt`) can be any text
 1. Use the following command to create the new secret.
 
   ```bash
-  dcos security secrets create -f my-secret.txt developer/my-secret
+  dcos security secrets create -t my-secret.txt developer/my-secret
```

Comment:
This is a backwards incompatible change, right? Maybe that should be noted?

Reply:
It is a backward incompatible change and I am not sure where to note it. We don't have any release notes section for the enterprise CLI changes.

CC @bamarni